content/toc-doc.json

```diff
@@ -378,15 +378,61 @@
         "slug": "/docs/tina-cloud/dashboard",
         "title": "Dashboard"
       },
-      {
-        "id": "/docs/tina-cloud/client",
-        "slug": "/docs/tina-cloud/client",
-        "title": "Client"
-      },
       {
         "id": "/docs/tina-cloud/cli",
         "slug": "/docs/tina-cloud/cli",
-        "title": "Command Line Interface"
+        "title": "Tina Cloud CLI",
+        "items": [
+          {
+            "id": "/docs/tina-cloud/cli/#defineschema",
+            "slug": "/docs/tina-cloud/cli/#defineschema",
+            "title": "Defining a schema",
+            "items": [
+              {
+                "id": "/docs/tina-cloud/cli/#collections",
+                "slug": "/docs/tina-cloud/cli/#collections",
+                "title": "Collections"
+              },
+              {
+                "id": "/docs/tina-cloud/cli/#templates",
+                "slug": "/docs/tina-cloud/cli/#templates",
+                "title": "Templates"
+              },
+              {
+                "id": "/docs/tina-cloud/cli/#fields",
+                "slug": "/docs/tina-cloud/cli/#fields",
+                "title": "Fields"
+              }
+            ]
+          },
+          {
+            "id": "/docs/tina-cloud/cli/#run-the-local-graphql-server",
+            "slug": "/docs/tina-cloud/cli/#run-the-local-graphql-server",
+            "title": "Local server"
+          }
+        ]
+      },
+      {
+        "id": "/docs/tina-cloud/client",
+        "slug": "/docs/tina-cloud/client",
+        "title": "Tina Cloud client",
+        "items": [
+          {
+            "id": "/docs/tina-cloud/client/#usegraphqlforms",
+            "href": "/docs/tina-cloud/client/#usegraphqlforms",
```

Comment:
@dwalkr this is reverting back to the global menu. I can't quite tell which key I'm supposed to use here

Reply:
@jeffsee55 url matching wasn't playing nice with hash URLs. pushed up a fix to this branch.